spec/factories/user_factory.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+
+  factory :user do
+    first_name "Foo"
+    last_name "Bar"
+    email "foo@bar.com"
```

Comment:
Consider creating a unique email for every user to meet uniqueness constraint in model